x-pack/plugins/lens/common/types.ts

```diff
@@ -57,8 +57,7 @@ export type CustomPaletteParamsConfig = CustomPaletteParams & {
 
 export type LayerType = typeof layerTypes[keyof typeof layerTypes];
 
-// Shared by XY Chart and Heatmap as for now
-export type ValueLabelConfig = 'hide' | 'inside' | 'outside';
+export type ValueLabelConfig = 'hide' | 'show';
```

Comment:
It seems like this is affecting the heatmap visualization too - looks like we need to do the same migration over there?

Reply:
I've checked. It is not used in the heatmap. So, I've removed the comment too.

Reply:
https://github.com/elastic/kibana/blob/4e9a1d5ed80b587efbe261827d7b007fe0c44561/src/plugins/chart_expressions/expression_heatmap/common/expression_functions/heatmap_legend.ts#L26

Reply:
https://github.com/elastic/kibana/blob/4e9a1d5ed80b587efbe261827d7b007fe0c44561/x-pack/plugins/lens/public/heatmap_visualization/visualization.tsx#L339

Reply:
Ah right, I missed this change: https://github.com/elastic/kibana/pull/128255/files#diff-251ce8bc0a27bef75b31d95a8e0351721650f621808e88e8b11593a32101f156R65